llvm/include/llvm/Debuginfod/Debuginfod.h

```diff
@@ -46,6 +46,10 @@ bool canUseDebuginfod();
 /// environment variable.
 SmallVector<StringRef> getDefaultDebuginfodUrls();
 
+/// Sets the list of debuginfod server URLs to query. This overrides the
+/// environment variable DEBUGINFOD_URLS.
+void setDefaultDebuginfodUrls(SmallVector<StringRef> URLs);
```

Comment:
pass in a const reference to avoid copying on calling this?:
```
void setDefaultDebuginfodUrls(const SmallVector<StringRef> &URLs);
```

Reply:
Done.